### PR TITLE
add nfc tag model and service

### LINKED ISSUE #58: Create feathers service for nfc-tags
It would be nice to have a feathers service to CRUD nfc-tags.

### PR DESCRIPTION
**Pull request by julianhahn**
Closes #58

### CODE REVIEW

File: packages/backend/src/services/nfc-tags/nfc-tag.service.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  constructor(options: Partial<NedbServiceOptions>, app: Application) {
+    super(options);
+  }
+}
+
+export default (app: Application): void => {
+  const options = {
+    Model: createModel(app),
+    paginate: app.get('paginate'),
```
Comment (JanEberlein) [inline packages/backend/src/services/nfc-tags/nfc-tag.service.ts:24]:
```suggestion
```
We don't need paginate here

**Review by tcoro — APPROVED** [review]
looks fine to me

**Review by JanEberlein — APPROVED** [review]

File: packages/backend/src/services/nfc-tags/nfc-tag.service.ts (hunk 1 above)
Comment (anbraten) [reply packages/backend/src/services/nfc-tags/nfc-tag.service.ts]: Why not?

File: packages/backend/src/services/nfc-tags/nfc-tag.service.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+class NFCTagService extends Service<NFCTag> {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  constructor(options: Partial<NedbServiceOptions>, app: Application) {
+    super(options);
+  }
+}
+
+export default (app: Application): void => {
+  const options = {
+    Model: createModel(app),
```
Comment (anbraten) [inline packages/backend/src/services/nfc-tags/nfc-tag.service.ts:23]:
```suggestion
    Model: createModel(app),
    paginate: app.get('paginate'),
```

**Review by anbraten — APPROVED** [review]

**Pull request merged**